crates/vm/levm/src/gas_cost.rs

```diff
@@ -200,6 +200,18 @@ pub const POINT_EVALUATION_COST: u64 = 50000;
 
 pub const BLAKE2F_ROUND_COST: u64 = 1;
 
+pub const MULTIPLIER: u64 = 1000;
```

Comment:
This constant will be used also for the G2 MSM.
```suggestion
pub const BLS12_381_MSM_MULTIPLIER: u64 = 1000;
```